liquibase-integration-tests/src/test/java/liquibase/dbtest/AbstractIntegrationTest.java

```diff
@@ -183,28 +192,26 @@ public void setUp() throws Exception {
         // Do not count the test as successful if we skip it because of a failed login. Count it as skipped instead.
         org.junit.Assume.assumeTrue(database != null);
 
-        if (database != null) {
```

Comment:
The condition checked here is already assumed just two lines ago...

Reply:
Yep. Thanks for taking that duplicate out